Source/SIMPLib/Common/Constants.h

```diff
@@ -53,30 +65,29 @@
 namespace SIMPL
 {
 
-  typedef uint32_t Rgb;
-  const Rgb  RGB_MASK    = 0x00ffffff;                // masks RGB values
-  const QString PathSep("|");
-  static const uint8_t Unchecked = 0;
-  static const uint8_t PartiallyChecked = 1;
-  static const uint8_t Checked = 2;
-
-  enum InfoStringFormat
-  {
-    HtmlFormat = 0,
-//      JsonFormat,
-//      TextFormat,
-//      XmlFormat,
-    UnknownFormat
-  };
+using Rgb = uint32_t;
+const Rgb RGB_MASK = 0x00ffffff; // masks RGB values
+const QString PathSep("|");
+static const uint8_t Unchecked = 0;
+static const uint8_t PartiallyChecked = 1;
+static const uint8_t Checked = 2;
```

Comment:
`static const` global

Reply:
As we discussed, we are going to leave these for the time being until we can get on C++17 and use inline keyword